### PR TITLE
[FE#81] Move site address reference to configuration

### PR DESCRIPTION
**Pull request by boris-arkenaar**
closes Signalen/frontend#81

There was a reference left in the code to Amsterdam by means of the site's address. This PR moves this to the language section of the configuration `language.siteAddress`.

**Update to `app.base.json`**

`language.siteAddress`
The site's address, for having a means of referencing the site by its address.
Default value: `meldingen.amsterdam.nl`.

### CODE REVIEW

**Comment by jpoppe** [comment]
Nice, will pick this up later this week with @MauritsR since we also have to update our other configuration before we could merge this.

**Comment by boris-arkenaar** [comment]
@jpoppe For Amsterdam you don't have to change anything to the configuration for this PR to merge. The original value is the default value in the config!

**Review by aditudorache — DISMISSED** [review]
Will put this on hold for the moment. There are some changes required in our mono repo for this

**Comment by aditudorache** [comment]
I hvae checked and it looks that there are no additional changes needed for this in the mono repo.

**Review by aditudorache — APPROVED** [review]

**Pull request merged**
